system-tests/tests/smoke/cre/por_test.go

```diff
@@ -755,7 +757,11 @@ func setupTestEnvironment(t *testing.T, testLogger zerolog.Logger, in *TestConfi
 		Topology:      topology,
 	}
 
-	err = libcontracts.ConfigureKeystone(configureKeystoneInput, []keystonetypes.DONCapabilityWithConfigFactoryFn{libcontracts.DefaultCapabilityFactoryFn, libcontracts.ChainWriterCapabilityFactory(libc.MustSafeUint64(int64(chainIDInt)))})
+	capFunc := libcontracts.DefaultCapabilityFactoryFn
+	if capabilityFactoryFns != nil {
+		capFunc = capabilityFactoryFns
+	}
```

Comment:
instead of doing that let's explicitly pass the default function for most of the test? I think it will increase readability